DataFormats/SiStripDetId/interface/SiStripDetId.h

```diff
@@ -112,18 +112,18 @@ class SiStripDetId : public DetId {
 // ---------- inline methods ----------
```

Comment:
i find
return 1 == (id_>>sterStartBit_ ) & sterMask_ )  ? 1:0;

very clear and obvious